# map params
cell_size: 0.2 # meters
global_map_size: 5.0 # meters
local_map_size: 5.0 # meters
grid_map_frame_id: Archimede_foot_start
elevation_logic: percentile # choose mean or percentile (median -> percentile 0.5)
percentile: 0.8

# layer names
elevation_layer_name: elevation
elevation_variance_layer_name: elevation_variance
cloudPoint_counter_layer_name: cloudPoint_counter
obstacles_layer_name: obstacles
obstacles_variance_layer_name: obstacles_variance
surface_orientation_layer_name: surface_orientation
surface_orientation_x_layer_name: surface_orientation_x
surface_orientation_y_layer_name: surface_orientation_y
surface_orientation_z_layer_name: surface_orientation_z
surface_orientation_variance_layer_name: surface_orientation_variance

# Archimede robot params
foot_print_frame_id: Archimede_footprint
max_slope_threshold: 0.349066 # 20 deg in radiants

# lidar params
point_cloud_max_height_threshold: 200 # meters
lidar_variance: 0.2
lidar_frame_id: Archimede_velodyne

# ros topics params
lidar_topic_name: /Archimede/velodyne_points
pc2_filtered_topic_name: /Archimede/velodyne_points_filtered
odom_topic_name: /gazebo_2_odom
grid_map_update_topic_name: /grid_map_update
grid_map_topic_name: /grid_map
grid_map_global_update_topic_name: /grid_map_global_update
